Adding myself to the LGPL list

douggeiger opened the pull request:
I, Douglas Geiger, hereby resubmit all my contributions to the VOLK project and repository under the terms of the LGPLv3+. My GitHub handle is douggeiger, my email addresses used for contributions are: doug.geiger@bioradiation.net

douggeiger committed:
Adding myself to the LGPL list

I, Douglas Geiger, hereby resubmit all my contributions to the VOLK project and repository under the terms of the LGPLv3+. My GitHub handle is douggeiger, my email addresses used for contributions are: doug.geiger@bioradiation.net

michaelld reviewed (changes requested):
Thanks, Doug! Would you also please add your other email used: `douglas.geiger@nrl.navy.mil` ? Just being complete ...

douggeiger committed:
Updating commit with additional email address

I, Douglas Geiger, hereby resubmit all my contributions to the VOLK project and repository under the terms of the LGPLv3+. My GitHub handle is douggeiger, my email addresses used for contributions are: doug.geiger@bioradiation.net, douglas.geiger@nrl.navy.mil

douggeiger commented:
> Thanks, Doug! Would you also please add your other email used: `douglas.geiger@nrl.navy.mil` ? Just being complete ...

Sure - updated pull request

michaelld reviewed (approved):
LGTM; thx!
